docs/extension/guides/set-wallet-recovery.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: '🦊 Set Wallet Recovery'
+sidebar_position: 6
+description: 'Extension Support: How to set up wallet recovery'
```

Comment:
This text will be avail. here on search engines:

<img width="612" alt="image" src="https://github.com/lukso-network/website-support-center/assets/477945/8e2f5732-1118-48d9-8c07-9002afdef18d">

So humans will read it.

```suggestion
description: 'How to set up wallet recovery on the LUKSO Universal Profile Extension?'
```

or something like this